es6-operators/main.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
```

Comment:
This says "execute an expression" but you instead, "assigned a value". Try again!